.github/workflows/build.yml

```diff
@@ -1,5 +1,5 @@
 name: HAL
-on: [push, pull_request]
+on: [push]
```

Comment:
Does this not disable the trigger for pull requests coming from outside the org's repo?

Reply:
Possible. Need to investigate, maybe we have to filter out push request triggers for main repository.
It's a good catch. I'm so used to working on one repo w/o forks.

Reply:
Does the trigger overlap cause any significant overhead? I can't recall. In either case, this feels like a common use case: perhaps there is a flag that does this for us?

Reply:
I think [2d75357](https://github.com/atsams-rs/atsamx7x-rust/pull/64/commits/2d75357e2be52ea1b2ecaa3e0eb9e7132b34e5e9) does the job, for 100% certainty we'd need someone PRing from the for after we merge this.